filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/batch/GcsBatchCommandBuilder.scala

```diff
@@ -34,4 +42,14 @@ trait GcsBatchCommandBuilder extends DefaultIoCommandBuilder {
     case gcsPath: GcsPath => GcsBatchExistsCommand(gcsPath)
     case _ => super.existsCommand(path)
   }
+
+  override def writeCommand(path: Path, content: String, options: OpenOptions) = path match {
+    case ossPath: OssPath => OssIoWriteCommand(ossPath, content, options)
+    case _ => super.writeCommand(path, content, options)
+  }
+
+  override def contentAsStringCommand(path: Path) = path match {
+    case ossPath: OssPath => OssIoContentAsStringCommand(ossPath)
+    case _ => super.contentAsStringCommand(path)
```

Comment:
Same here, now that you implemented the nio interface you stick to `DefaultIoCommand`s and it will all work